### DISCUSSION TITLE
Changing mode from "onBlur" to "onChange" after first error

### DISCUSSION THREAD

**Discussion by pgsandstrom** [react-hook-form/react-hook-form#4872] [Q&A] [answered]
So I want errors to show on "onBlur", but after the error is displayed I want it to go away as soon as the input is correct, i.e. I want "onChange".

Unfortunately "reValidateMode" doesn't seem to help me, since that is only used after submit. How can I solve this in the neatest way possible?

**Comment by bluebill1049** [comment, marked as answer]
mode: 'onTouched' ?
